iquip/apps/builder.py

```diff
@@ -291,13 +291,16 @@ def value(self) -> Optional[str]:
         return None
 
 
-# TODO(AIJUH): Add feature for argInfo processing and other scan type classes.
+# TODO(AIJUH): Add other scan type classes.
 class _ScanEntry(QWidget):
```

Comment:
I have a question. Why did you inherit from `QWidget` directly, not `_BaseEntry`?

Reply:
I see. There seems to be several differences between `_BaseEntry` and `_ScanEntry`.